strawberry/cli/commands/server.py

```diff
@@ -57,4 +57,4 @@ def server(schema, host, port, app_dir):
         app.add_websocket_route(path, graphql_app)
 
     print(f"Running strawberry on http://{host}:{port}/ 🍓")
-    uvicorn.run(app, host=host, port=port, log_level="error")
+    uvicorn.run(app, loop="none", host=host, port=port, log_level="error")
```

Comment:
Is this documented in the uvicorn docs? I was only able to find it here: https://github.com/encode/uvicorn/blob/bf1c64e2c141971c546671c7dc91b8ccf0afeb7d/uvicorn/config.py#L55-L60

I'll test this with the dataloader example and see if it works :)

Thank you!

Reply:
It appears to be undocumented on their website which matches the uvicorn cli help command. Maybe it was intended for programmatically usage only. However, according to [this PR](https://github.com/encode/uvicorn/pull/455) they added it specifically to allow the setup of custom event loops.